src/Resources/views/CRUD/base_acl_macro.html.twig

```diff
@@ -10,13 +10,10 @@ file that was distributed with this source code.
 #}
 
 {% macro render_form(form, permissions, td_type, admin, admin_configuration, object) %}
-    <form class="form-horizontal"
-          action="{{ admin.generateUrl('acl', {(admin.idParameter): admin.id(object), 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}"
-          {% if form.vars.multipart %} enctype="multipart/form-data"{% endif %}
```

Comment:
this is missing

Reply:
This is not needed, because symfony forms will add this